File: tests/fixtures/sample.ts
import { Request, Response } from 'express';
import { db } from './database';

export class UserController {
  async getUser(req: Request, res: Response) {
    const user = await db.findUser(req.params.id);
    res.json(user);
  }

  async deleteUser(req: Request, res: Response) {
    await db.deleteUser(req.params.id);
    res.status(204).send();
  }
}

export function validateInput(data: unknown): boolean {
  return typeof data === 'string' && data.length > 0;
}

const helper = (x: number) => x * 2;

File: tests/fixtures/sample.py
from flask import request, jsonify
from models import User

class UserService:
    def get_user(self, user_id):
        user = User.query.get(user_id)
        return user

    def delete_user(self, user_id):
        user = User.query.get(user_id)
        user.delete()

def validate_input(data):
    return isinstance(data, str) and len(data) > 0
